languages/react-native/example/ios/Podfile

```
@@ -0,0 +1,43 @@
+ENV['RCT_NEW_ARCH_ENABLED'] = '1'
+
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
```

RuboCop (1.75.5)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)